lib/core/src/model.rs

```diff
@@ -132,6 +130,31 @@ impl Config {
         }
     }
 
+    pub fn mainnet_esplora(breez_api_key: Option<String>) -> Self {
```

Comment:
The use of `mainnet_esplora()` depends Wasm, so why not switch the function depending on Wasm. Same for the other networks
```suggestion
    #[cfg(all(target_family = "wasm", target_os = "unknown"))]
    pub fn mainnet(breez_api_key: Option<String>) -> Self {
```

Reply:
I think it may be useful to have an esplora default available on native builds as well, so we could even rename the mainnet to mainnet_electrum. If not for anything else, for the CLI and end-to-end tests. But I'm not strongly opinionated here.

Reply:
Sorry for not explaining, it was mostly an online conversation with @danielgranhao. Opening up the mainnet_esplora also allows non-wasm clients to select that branch if they wish to

Reply:
No worries 👍